zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -2865,6 +2849,7 @@ components:
       type: object
       properties:
         items:
+          description: sjh
```

Comment:
Each of these is a fake description placeholder, which I will fill in as part of this PR.